conf/setup.routes

```diff
@@ -0,0 +1,4 @@
+# Data setup routes
+
+POST    /data             @controllers.SetupDataController.addData
+DELETE  /all-data         @controllers.SetupDataController.removeAll
```

Comment:
Is there meant to be a route for removeData as well or will that only be used internally to the app?

Reply:
We've never used it before in our other dynamic stubs, and I don't see a plan to use it here so I'll remove the function.

Reply:
OK